icd/api/include/color_space_helper.h

```diff
@@ -55,17 +55,20 @@ class ColorSpaceHelper
         Fmt_8bpc_srgb    = 0x0008,
         Fmt_8bpc_unorm   = 0x0010,
         Fmt_9bpc         = 0x0020,
-        Fmt_10bpc        = 0x0040,
-        Fmt_11bpc        = 0x0080,
-        Fmt_12bpc        = 0x0100,
-        Fmt_16bpc_unorm  = 0x0200,
-        Fmt_16bpc_sfloat = 0x0400,
-        Fmt_32bpc        = 0x0800,
-
-        Fmt_8bpc     = Fmt_8bpc_srgb   | Fmt_8bpc_unorm,
-        Fmt_16bpc    = Fmt_16bpc_unorm | Fmt_16bpc_sfloat,
-        Fmt_KnownHDR = Fmt_10bpc | Fmt_11bpc | Fmt_12bpc | Fmt_16bpc,
-        Fmt_All      = Fmt_4bpc  | Fmt_5bpc  | Fmt_6bpc  | Fmt_8bpc  | Fmt_KnownHDR | Fmt_32bpc,
+        Fmt_10bpc_srgb   = 0x0040,
+        Fmt_10bpc_unorm  = 0x0080,
+        Fmt_11bpc        = 0x0100,
+        Fmt_12bpc        = 0x0200,
+        Fmt_16bpc_unorm  = 0x0400,
+        Fmt_16bpc_sfloat = 0x0800,
+        Fmt_32bpc        = 0x1000,
+
+        Fmt_8bpc      = Fmt_8bpc_srgb   | Fmt_8bpc_unorm,
+        Fmt_10bpc     = Fmt_10bpc_srgb  | Fmt_10bpc_unorm,
+        Fmt_16bpc     = Fmt_16bpc_unorm | Fmt_16bpc_sfloat,
+        Fmt_KnownHDR  = Fmt_10bpc | Fmt_11bpc | Fmt_12bpc | Fmt_16bpc,
```

Comment:
Note that 10-bit sRGB, while technically invalid for `TfHlg` (where this `KnownHDR` format is used), is still HDR. Should we change `TfHlg` to use a new `Fmt`, without the `srgb` variant?

Reply:
Thanks for your information @MarijnS95 . Could you help change "Fmt_KnownHDR = Fmt_10bpc_unorm| Fmt_11bpc | Fmt_12bpc | Fmt_16bpc" in this commit? Thanks.

Reply:
@JoeWangAMD sounds good! We can also decide to rename it to `Fmt_KnownHLG`, is that more clear or would you prefer to stick with `KnownHDR`?

Reply:
@JoeWangAMD I've updated this and also added `Fmt_10bpc_srgb` to `Fmt_All` to complement this change.

I haven't renamed it to HLG as this flag is also used in the `IsFmtHdr()` function.